packages/authorize/useAuthorize.js

```diff
@@ -105,6 +104,7 @@ export default (
     ) {
       setLoading(false);
       setAuthorized(authorized);
+      setCurrentRegion(currentRegion);
```

Comment:
I don't think we should set the current region conditionally. if it resolves, we should set it on line 84.